docs/book/v3/cookbook/access-common-data-in-templates.md

```diff
@@ -0,0 +1,75 @@
+# How Can I Access Common Data In Templates?
+
+This is a question that's asked a lot of times. How can I make common data like
+request attributes, the current route name, etc. available into all template.
+The answer is pretty easy actually. All that is needed is a middleware and
```

Comment:
Avoid words and phrases like "easy", "actually", "simply", "simple", "just", "all that is needed", etc. What is easy for the author is often difficult for the person looking for a solution, which is why they're reading the document in the first place.